website/docs/docs/dbt-cloud-apis/metadata-use-case-guides.md

```diff
@@ -0,0 +1,1138 @@
+          }
+        ]
+      },
+      {
+        "name": "stg_supplies",
+        "uniqueId": "model.demo_data.stg_supplies",
+        "dependsOn": [
+          "source.demo_data.ecommerce.supplies",
+          "macro.dbt_utils.surrogate_key"
+        ],
+        "parentsModels": [],
+        "parentsSources": [
+          {
+            "name": "supplies",
+            "uniqueId": "source.demo_data.ecommerce.supplies"
+          }
+        ]
+      },
+      {
+        "name": "dbt_metrics_default_calendar",
+        "uniqueId": "model.metrics.dbt_metrics_default_calendar",
+        "dependsOn": [
+          "macro.metrics.metric_date_spine",
+          "macro.dbt.date_trunc"
+        ],
+        "parentsModels": [],
+        "parentsSources": []
+      }
+    ],
+    "metrics": [
+      {
+        "name": "customers",
+        "uniqueId": "metric.demo_data.customers",
+        "dependsOn": [
+          "model.demo_data.orders"
+        ],
+        "model": {
+          "name": "orders",
+          "uniqueId": "model.demo_data.orders",
+          "dependsOn": [
+            "model.demo_data.stg_orders",
+            "model.demo_data.stg_order_items",
+            "model.demo_data.stg_products",
+            "model.demo_data.stg_locations",
+            "model.demo_data.stg_supplies"
+          ]
+        }
+      },
+      {
+        "name": "expenses",
+        "uniqueId": "metric.demo_data.expenses",
+        "dependsOn": [
+          "model.demo_data.orders"
+        ],
+        "model": {
+          "name": "orders",
+          "uniqueId": "model.demo_data.orders",
+          "dependsOn": [
+            "model.demo_data.stg_orders",
+            "model.demo_data.stg_order_items",
+            "model.demo_data.stg_products",
+            "model.demo_data.stg_locations",
+            "model.demo_data.stg_supplies"
+          ]
+        }
+      },
+      {
+        "name": "gross_profit",
+        "uniqueId": "metric.demo_data.gross_profit",
+        "dependsOn": [
+          "metric.demo_data.revenue",
+          "metric.demo_data.expenses"
+        ],
+        "model": null
+      },
+      {
+        "name": "revenue",
+        "uniqueId": "metric.demo_data.revenue",
+        "dependsOn": [
+          "model.demo_data.orders"
+        ],
+        "model": {
+          "name": "orders",
+          "uniqueId": "model.demo_data.orders",
+          "dependsOn": [
+            "model.demo_data.stg_orders",
+            "model.demo_data.stg_order_items",
+            "model.demo_data.stg_products",
+            "model.demo_data.stg_locations",
+            "model.demo_data.stg_supplies"
+          ]
+        }
+      }
+    ]
+```
+</TabItem>
+</Tabs>
+
+## Quality
+
+Information from dbt runs helps data teams monitor quality and data consumers know that they’re using fresh and accurate data for their analyses, mitigating the risk of people making the wrong decisions based on outdated or incorrect data.  
```

Comment:
does this mean 'dbt job runs'?

```suggestion
Information from dbt runs helps data teams monitor quality and data consumers know that they’re using fresh and accurate data for their analyses, mitigating the risk of people making the wrong decisions based on outdated or incorrect data.  
```

Reply:
I think "dbt runs" is unambiguous! If we want to make it more specific, we can say "dbt runs on dbt Cloud"